src/Server/Resource/DoctrineResource.php

```diff
@@ -330,7 +332,8 @@ public function create($data)
             return $data;
         }
 
-        $entity = new $entityClass;
+        $entity = (null !== $entityFactory) ? $entityFactory->get($entityClass) : new $entityClass;
```

Comment:
@TomHAnderson I created it as an external library because that was what you suggested (and I understand that). Would this PR get merged faster if we abandon that idea and move the library into this project? 

If not, could you maybe suggest a few things we should do for the library to be considered mature? We just want to push this forward if possible..